CK2Plus_expanded/history/provinces/1577 - Cinarca.txt

```diff
@@ -1,25 +1,25 @@
-# 1600 - Cinarca
-
-# County Title
-title = c_cinarca
-
-# Settlements
-max_settlements = 3
-b_cinarca = castle
-b_ajaccio = city
-b_bonifacio = temple
-#b_sartene = city
-#b_portevecchio = castle
-#b_filitosa = castle
-#b_bastelicaccia = temple
-#b_propriano = castle
-
-# Misc
-culture = corsican
-religion = catholic
-terrain = mountain
-
-# History
-1077.1.1 = { trade_post = b_della_gherardesca }
-1100.1.1 = { culture = italian }
-1195.1.1 = { trade_post = b_spinola }
+# 1600 - Cinarca
+
+# County Title
+title = c_cinarca
+
+# Settlements
+max_settlements = 3
+b_cinarca = castle
+b_ajaccio = city
+b_bonifacio = temple
+#b_sartene = city
+#b_portevecchio = castle
+#b_filitosa = castle
+#b_bastelicaccia = temple
+#b_propriano = castle
+
+# Misc
+culture = italian #sardinian

```

Comment:
This is what I was referring to in chat. The comment makes it sound like the intended culture was sardinian.

Reply:
Okay... wrong commit, this was the old stuff.